bosh-release/deploy.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
```

Comment:
Is this file being removed and added again as a new file to pattern-1/bosh-release/deploy.sh?

Reply:
@imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

Reply:
The PR shows some files have been moved but some as deleted and added as new.